test/test_ops.py

```diff
@@ -741,6 +741,18 @@ def test_backward(self, device, dtype, op):
             kwargs = sample.kwargs
             composite_compliance.check_backward_formula(op, args, kwargs)
 
+    @unittest.skipIf(IS_FBCODE or IS_SANDCASTLE, '__torch_dispatch__ does not work in fbcode')
+    @ops([op for op in op_db if op.supports_autograd], allowed_dtypes=(torch.float,))
+    def test_forward_ad(self, device, dtype, op):
```

Comment:
A more descriptive test name like "test_forward_ad_composite_compliance" would be nice

Reply:
The test case is already called TestCompositeCompliance, so this test reads as TestCompositeCompliance.test_forward_ad

Reply:
aha, sounds good